src/common/cmdparse.h

```diff
@@ -94,6 +107,19 @@ bool cmd_getval(
   }
 }
 
+template <typename T>
+std::optional<T> cmd_getval(const cmdmap_t& cmdmap,
+			    const std::string& k,
```

Comment:
might want to use `std::string_view` ?

Reply:
Well, this is just a wrapper which requires `std::string`.

Reply:
okay, will create a PR to fix this.

Reply:
#41434